ERROR - 2024-02-05 14:17:36 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\laragon\www\pos\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\pos\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `ti...', 0)
#1 C:\laragon\www\pos\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ti...')
#2 C:\laragon\www\pos\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ti...')
#3 C:\laragon\www\pos\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ti...', Array, false)
#4 C:\laragon\www\pos\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\pos\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\pos\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\pos\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\pos\app\Controllers\Ventas\TipoDocumento.php(168): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\pos\system\CodeIgniter.php(942): App\Controllers\Ventas\TipoDocumento->save()
#10 C:\laragon\www\pos\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas\TipoDocumento))
#11 C:\laragon\www\pos\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\pos\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-02-05 14:17:36 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tipo_documento` (`descripcion`, `estado`, `created_at`, `updated_at`) VALUES (:descripcion:, :estado:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Ventas\TipoDocumento.php(168): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Ventas\TipoDocumento->save()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas\TipoDocumento))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-05 14:19:17 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\laragon\www\pos\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\pos\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `ti...', 0)
#1 C:\laragon\www\pos\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ti...')
#2 C:\laragon\www\pos\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ti...')
#3 C:\laragon\www\pos\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ti...', Array, false)
#4 C:\laragon\www\pos\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\pos\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\pos\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\pos\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\pos\app\Controllers\Ventas\TipoDocumento.php(168): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\pos\system\CodeIgniter.php(942): App\Controllers\Ventas\TipoDocumento->save()
#10 C:\laragon\www\pos\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas\TipoDocumento))
#11 C:\laragon\www\pos\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\pos\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-02-05 14:19:17 --> Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tipo_documento` (`descripcion`, `estado`, `created_at`, `updated_at`) VALUES (:descripcion:, :estado:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Ventas\TipoDocumento.php(168): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Ventas\TipoDocumento->save()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas\TipoDocumento))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-05 12:08:11 --> Undefined array key "updated_at"
in APPPATH\Controllers\Ventas\TipoDocumento.php on line 143.
 1 APPPATH\Controllers\Ventas\TipoDocumento.php(143): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "updated_at"', 'C:\\laragon\\www\\pos\\app\\Controllers\\Ventas\\TipoDocumento.php', 143)
 2 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Ventas\TipoDocumento->ajaxListarTipoDocumento()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas\TipoDocumento))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
